src/constants.js

```diff
@@ -0,0 +1,8 @@
+const proxy = 'https://immense-hamlet-20285.herokuapp.com/';
+const api = 'https://pfa.foreca.com';
+export const url = proxy + api;
+
+export const authData = {
+  user: 'nmakeychik1',
+  password: '64vwAHlJXbJN'
```

Comment:
nice, right now I can use your credentials. Thank you)

Reply:
moved them to .env on the server side